torchmetrics/text/wer.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any
```

Comment:
remember to include license in the top
https://github.com/PyTorchLightning/metrics/blob/5c2069eb2087ede7f008222ebab286aedad14dfb/torchmetrics/utilities/imports.py#L1-L13

Reply:
Added